discovery-provider/src/api/v1/helpers.py

```diff
@@ -217,6 +217,8 @@ def extend_playlist(playlist):
     playlist["added_timestamps"] = add_playlist_added_timestamps(playlist)
     playlist["cover_art"] = playlist["playlist_image_multihash"]
     playlist["cover_art_sizes"] = playlist["playlist_image_sizes_multihash"]
+    playlist["track_count"] = (playlist["track_count"] if "track_count"
```

Comment:
maybe add a comment here on why we would sometimes already have "track_count"

Reply:
will do